config/features/flags.go

```diff
@@ -180,7 +180,5 @@ var BeaconChainFlags = append(deprecatedFlags, []cli.Flag{
 // E2EBeaconChainFlags contains a list of the beacon chain feature flags to be tested in E2E.
 var E2EBeaconChainFlags = []string{
 	"--dev",
-	"--use-check-point-cache",
```

Comment:
This is not present in `deprecated_flags.go`

Reply:
It hasn't been used in 2 years and the feature is a no-op currently. So it doesn't actually exist.